odl/tomo/geometry/spect.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2014 - 2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Single-photon emission computed tomography (SPECT) geometry."""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import super
+
+import numpy as np
+
+from odl.tomo.geometry.geometry import AxisOrientedGeometry
+from odl.tomo.geometry.parallel import Parallel3dAxisGeometry
+from odl.tomo.util.utility import perpendicular_vector
+
+__all__ = ('ParallelHoleCollimatorGeometry', )
+
+
+class ParallelHoleCollimatorGeometry(Parallel3dAxisGeometry):
+
+    """Geometry for SPECT Parallel hole collimator."""
+
+    def __init__(self, apart, dpart, det_rad, axis=[0, 0, 1],
+                 **kwargs):
+        """Initialize a new instance.
+        Parameters
+        ----------
+        apart : 1-dim. `RectPartition`
+            Partition of the angle interval
+        dpart : 2-dim. `RectPartition`
+            Partition of the detector parameter rectangle
+        det_rad : positive `float`
+            Radius of the circular detector orbit.
+        axis : `array-like`, shape ``(3,)``, optional
+            Fixed rotation axis.
+        orig_to_det_init : `array-like`, shape ``(3,)``, optional
+            Vector pointing towards the detector reference point in
+            the initial position. The zero vector is only allowed if
+            ``det_init_axes`` is explicitly given.
+            Default: a `perpendicular_vector` to ``axis``.
+        det_init_axes : 2-tuple of `array-like` (shape ``(3,)``), optional
+            Initial axes defining the detector orientation.
+            Default: the normalized cross product of ``axis`` and
+            ``det_init_pos`` is used as first axis and ``axis`` as second.
```

Comment:
No, I think you understood me correctly. I commented under the exact line I meant, not 3 lines up.